src/twisted/internet/test/test_inlinecb.py

```diff
@@ -128,20 +128,71 @@ class ForwardTraceBackTests(SynchronousTestCase):
 
     def test_forwardTracebacks(self):
         """
-        Test that chained inlineCallbacks are properly forwarding the traceback informations
-        from generator to generator
-        Reproducing #9175
+        Chained inlineCallbacks are forwarding the traceback informations
```

Comment:
I have pushed a commit for fixing the information*s* typo, but I see that it was lost.

Reply:
sorry. I fixed that.